packages/kmod-6.12-nvidia-r570/kmod-6.12-nvidia-r570.spec

```diff
@@ -179,9 +182,8 @@ install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/{drivers,ld.so.c
 install -d %{buildroot}%{_cross_sysusersdir}
 install -d %{buildroot}%{_cross_bindir}
 
-KERNEL_VERSION=$(cat %{kernel_sources}/include/config/kernel.release)
 sed \
-  -e "s|__KERNEL_VERSION__|${KERNEL_VERSION}|" \
+  -e "s|__KERNEL_VERSION__|%{kernel_major}|" \
```

Comment:
I don't believe the path we are swapping out for this has the symlink we would need here, we need the M.m.p for the kernel, not just M.m.
```
ls /x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/modules
6.12.20
```
From `nvidia-tmpfiles.conf.in`:
```
nvidia-tmpfiles.conf.in
1:R __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/tesla - - - - -
2:d __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/tesla 0755 root root - -
3:R __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/open-gpu - - - - -
4:d __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/open-gpu 0755 root root - -
```

Reply:
@bcressey added a change in #99 to install a symlink using just the major version.

Reply:
Oh nice! I missed this change in the other PR.